hcn/hcnsupport.go

```diff
@@ -81,16 +140,10 @@ func GetSupportedFeatures() SupportedFeatures {
 	features.NetworkACL = isFeatureSupported(globals.Version, NetworkACLPolicyVersion)
 	features.NestedIpSet = isFeatureSupported(globals.Version, NestedIpSetVersion)
 
-	// Only print the HCN version and features supported once, instead of everytime this is invoked. These logs are useful to
-	// debug incidents where there's confusion on if a feature is supported on the host machine. The sync.Once helps to avoid redundant
-	// spam of these anytime a check needs to be made for if an HCN feature is supported. This is a common occurrence in kubeproxy
-	// for example.
-	versionOnce.Do(func() {
-		logrus.WithFields(logrus.Fields{
-			"version":           fmt.Sprintf("%+v", globals.Version),
-			"supportedFeatures": fmt.Sprintf("%+v", features),
-		}).Info("HCN feature check")
-	})
+	logrus.WithFields(logrus.Fields{
```

Comment:
Shouldn't the existing function call GetCachedSupportedFeatures and just swallow the error?

Reply:
I agree, though we should probably log the error instead of swallowing completely.

Reply:
Sure, I wanted to leave the original function alone mostly was my thought process

Reply:
Done

Reply:
Oh Kevins comment didn't show up the first time.. that's weird. It's being logged currently but at a debug level and via logrus so generally it won't show up unless the target application is also using logrus and has the level set to allow these.. Possibly bump it to error level?